docs/wasm/sc-dev/use-openbrush.md

```diff
@@ -33,25 +33,22 @@ Which standard tokens and useful contracts does it provide?
 
 ### Generic Trait Implementation
 
-More importantly, OpenBrush adds support for generic Trait implementation so you can split Trait and Impls into different files, which will increase the readability and maintainability of your smart-contract code base (see detailed description [here](https://github.com/Supercolony-net/openbrush-contracts/blob/c71097ff10ef6d187bf9c86f0248bfbc6ec40a3c/docs/docs/smart-contracts/example/setup_project.md))
+More importantly, OpenBrush adds support for generic Trait implementation so you can split Trait and Impls into different files, which will increase the readability and maintainability of your smart-contract code base (see detailed description [here](https://github.com/727-Ventures/openbrush-contracts/blob/main/docs/docs/smart-contracts/example/setup_project.md))
 
 ### Wrapper around Traits
 
 Defining a Trait definition is sufficient (a contract that implements that Trait is not needed anymore) to call methods of that Trait from some contract in the network (do a cross contract call). It makes cross-contract calls easier.
 
 ### Documentation
 
-- [OpenBrush Github repo](https://github.com/Supercolony-net/openbrush-contracts)
+- [OpenBrush Github repo](https://github.com/727-Ventures/openbrush-contracts)
 - [Official Docs](https://docs.openbrush.io/)
 - [OpenBrush website](https://openbrush.io/)
 - [Substrate Seminar (Youtube)](https://www.youtube.com/watch?v=I5OFGNVvzOc)
 - [How to use modifiers](https://medium.com/supercolony/how-to-use-modifiers-for-ink-smart-contracts-using-openbrush-7a9e53ba1c76)
 
-### `parity-common`
+You can join the [Brushfam Element channel](https://matrix.to/#/!utTuYglskDvqRRMQta:matrix.org?via=matrix.org&via=t2bot.io&via=web3.foundation) if you have any question about OpenBrush or ink! smart contracts in general.
```

Comment:
We generally don't allow adding SNS links directly to our docs. Could you add the link to the READ of the tools instead?